scripts/tests/browser_translation_test.js

```diff
@@ -85,9 +87,69 @@ add_task(async function testTranslationBarDisplayed() {
        );
     }
 
+    // check quality estimation
+    const checkQualityEstimation = async (document, message) => {
+      const translation = document.getElementById("translationDiv").innerText;
+      const translatedHTMLWithQEScores = document.getElementById("translationDiv").innerHTML;
+
+      // just check for the translated content before checking quality estimation
```

Comment:
this use case is already covered in another test, why repeat it here?

Reply:
Before going ahead with checking the quality scores of the translated content, I want to make sure that the translation actually happened. Otherwise, it is not apparent that qe test failures happened because translation failed.

Reply:
why? you will see that exactly the same "is" assertion failed in `checkTranslation` function

Reply:
Ideally, QE tests shouldn't really depend on what happens with `checkTranslation` function and it should remain self-sufficient.